systems/subscriber/subscriber-registry/pkg/server/subscriber.go

```diff
@@ -0,0 +1,288 @@
+package server
+
+import (
+	"context"
+
+	"github.com/gofrs/uuid"
+	"github.com/sirupsen/logrus"
+	"github.com/ukama/ukama/systems/common/grpc"
+	mb "github.com/ukama/ukama/systems/common/msgBusServiceClient"
+	"github.com/ukama/ukama/systems/common/msgbus"
+	pb "github.com/ukama/ukama/systems/subscriber/subscriber-registry/pb/gen"
+	"github.com/ukama/ukama/systems/subscriber/subscriber-registry/pkg"
+	"github.com/ukama/ukama/systems/subscriber/subscriber-registry/pkg/db"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type SubcriberServer struct {
+	subscriberRepo       db.SubscriberRepo
+	msgbus               mb.MsgBusServiceClient
+	subscriberRoutingKey msgbus.RoutingKeyBuilder
+	pb.UnimplementedSubscriberRegistryServiceServer
+}
+
+
+
+func NewSubscriberServer(subscriberRepo db.SubscriberRepo, msgBus mb.MsgBusServiceClient) *SubcriberServer {
+	return &SubcriberServer{subscriberRepo: subscriberRepo,
+		msgbus:         msgBus,
+		subscriberRoutingKey: msgbus.NewRoutingKeyBuilder().SetCloudSource().SetContainer(pkg.ServiceName)}
+}
+
+
+
+func (s *SubcriberServer) Add(ctx context.Context, req *pb.AddSubscriberRequest) (*pb.AddSubscriberResponse, error) {
+	logrus.Infof("Adding subscriber: %v", req)
+	networkID_uuid := uuid.FromStringOrNil(req.GetNetworkID())
```

Comment:
1) Use camelCase notation instead of snake_case.
2) Use either networkID or networkUUID, but not both.
3) What do we do when networkID is nil?